### DISCUSSION TITLE
!! REopt is experiencing intermittent server issues 11/26-11/27

### DISCUSSION THREAD

**Discussion by LParkhill** [NREL/REopt-API-Analysis#88] [Announcements]
The team has been alerted and will resolve these issues as soon as possible. Thanks for your patience!

**Comment by LParkhill** [comment]
These server issues are currently causing evaluations to fail with an "Unexpected Error" message. We apologize for the inconvenience.

**Comment by azharsq1** [comment]
Hi, Your prompt resolution of the issue will be appreciated. Thanks

**Reply by LParkhill** [reply]
@azharsq1   REopt issue has been resolved. Thanks for your patience.
